app-backend/build.sbt

```diff
@@ -117,3 +118,16 @@ lazy val database = Project("database", file("database"))
   .settings({
    libraryDependencies ++= slickDependencies ++ dbDependencies ++ loggingDependencies ++ Seq(Dependencies.akkaHttpExtensions)
   })
+
+lazy val tiler = Project("tileserver", file("tileserver"))
+  .settings(commonSettings:_*)
```

Comment:
Might I suggest putting this in the `tile-server/build.sbt` This is not very related to settings of other projects so moving it would keep the root `build.sbt` comprehensible and brief.